.github/workflows/test.yml

```diff
@@ -0,0 +1,14 @@
+name: "Test"
+on:
+  pull_request:
+  push:
+jobs:
+  tests:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cachix/install-nix-action@v8
+    - run: nix-build release.nix --arg nix '{ outPath = ./.; revCount = 123; shortRev = "abcdefgh"; }' --arg systems '[ builtins.currentSystem ]' -A build -A binaryTarball -A coverage -A perlBindings -A tarball -A installerScript
```

Comment:
This will not work once the flakes branch is merged because `flake.nix` (which replaces `release.nix`) does not accept arguments, and `builtins.currentSystem` does not exist.

Reply:
No need to build the `coverage` job. It's currently broken, takes a long time to build and nobody every looks at it anyway.

Reply:
> This will not work once the flakes branch is merged because `flake.nix` (which replaces `release.nix`) does not accept arguments, and `builtins.currentSystem` does not exist.


Why cant' the `release.nix` stay for compatibility with non-flake users? Couldn't we just do an import of `release.nix` in the flake?

Reply:
That's what I did originally, but it's much more convenient to have one file rather than a release.nix, shell.nix, release-common.nix, ...